apps/api_web/config/config.exs

```diff
@@ -28,7 +28,21 @@ config :api_web, :rate_limiter,
   limiter: ApiWeb.RateLimiter.ETS,
   max_anon_per_interval: 5_000,
   max_registered_per_interval: 100_000,
-  wait_time_ms: 0
+  wait_time_ms: 0,
+  connection_opts: [
+    coder: Memcache.Coder.JSON
+  ]
+
+config :api_web, :rate_limiter_concurrent,
+  enabled: false,
+  memcache: false,
```

Comment:
suggestion: enable the rate limiter by default. Then we can get rid of the calls enabling it in `dev` and `prod`, simplifying configuration and ensuring that we default to rate limiting.

Reply:
My only concern with that is it would then require memcached running locally - this feature was built with the intent of it being a global limiter, so there's no local-only implementation to fall back on.

Reply:
This should be fixed now, they should be disabled in `dev`.